cognite/neat/rules/models/dms/_validation.py

```diff
@@ -26,12 +27,17 @@ def __init__(self, rules: DMSRules):
         self.issue_list = IssueList()
 
     def validate(self) -> IssueList:
-        self._validate_best_practices()
+        self._validate_raw_filter()
         self._consistent_container_properties()
+
         self._referenced_views_and_containers_are_existing()
```

Comment:
Should we do any check if SchemaCompleteness is `partial` ?

Reply:
I think this method will break if .last is missing?

Reply:
I have this:
```
        if not self.rules.last:
            raise ValueError("The schema is set to 'extended', but no last rules are provided to validate against")
```